flows/actions/modifiers/status.go

```diff
@@ -0,0 +1,61 @@
+package modifiers
+
+import (
+	"encoding/json"
+
+	"github.com/nyaruka/goflow/assets"
+	"github.com/nyaruka/goflow/envs"
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/events"
+	"github.com/nyaruka/goflow/utils"
+)
+
+func init() {
+	registerType(TypeStatus, readStatusModifier)
+}
+
+// TypeStatus is the type of our status modifier
+const TypeStatus string = "status"
+
+const modifierActiveContactStatus flows.ContactStatus = "active"
```

Comment:
modifier will be using `"active"` but the `contact` will be using ""

We map that on

https://github.com/nyaruka/goflow/pull/906/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json&hide-deleted-files=true#diff-248bca69f6b915f47e0f442a92a70902R40-R43

Reply:
Not following.. we've said we want `contact` to have a `status` field the way `run` and `session` have..

```
@run.status = active|completed|...
@contact.status = active|blocked|...
```

Reply:
I get that now